test/books/details-public-book.test.js

```diff
@@ -0,0 +1,59 @@
+import { Selector } from 'testcafe';
+import { runBaseTests } from '../../br-e2e-tests/helpers/base.js';
+import { runRightToLeftTests } from '../../br-e2e-tests/helpers/rightToLeft.js';
+import BookReader from '../../br-e2e-tests/models/BookReader.js';
+import { DesktopNav } from '../../br-e2e-tests/models/Navigation.js';
+import { runSearchTests } from './helpers/search-inside.js';
+
+const localURL = 'https://archive.org/details/';
+const books = [
+  'birdbookillustra00reedrich', // publicDomain
+  // 'pianoservicingtu00rebl_0', // borrowable,
+  // 'adventuresoftoms00twaiiala', // has chapter markers
+  // 'gendaitankashu00meijuoft', // Right to Left book
+  // 'gov.uspto.patents.application.10074026', // multiple files,
+  // 'theworksofplato01platiala', //bookreader demo
```

Comment:
QUESTION: How are these commented-out lines used? If we are to keep commented out code, maybe we should explain why it's there and how to use it...?

Reply:
these books are not yet supported with their specific tests. good to keep them there for specific examples. i've added a comment.